### DISCUSSION TITLE
How to build the latest version into a Docker image?

### DISCUSSION THREAD

**Discussion by andy263** [toeverything/AFFiNE#2015] [Q&A] [answered]
The official image is still from three months ago.

**Comment by ghost** [comment]
Why is the docker image for AFFiNE so out of date? AFFiNE Cloud has so much more going for it in comparison.

I was extremely excited to use AFFiNE, but I stopped using it the moment I realized that the focus has been on AFFiNE Cloud.

Can we please get an official response on this?

**Reply by himself65** [reply, marked as answer]
We have been refactoring the AFFiNE Cloud for months because the old infrastructure cannot offer the best experience for our clients.

I think we could upgrade all of the stuff by the end of this month including docker image.

See: https://github.com/toeverything/AFFiNE/pull/2924

**Reply by ghost** [reply]
> We have been refactoring the AFFiNE Cloud for months because the old infrastructure cannot offer the best experience for our clients.
> 
> I think we could upgrade all of the stuff by the end of this month including docker image.
> 
> See: https://github.com/toeverything/AFFiNE/pull/2924

That sounds great! Thank you for the official response!
